R/tfrmt.R

```diff
@@ -43,6 +43,11 @@
 #'   to `column` as "Hello World". To pass "my_object" to tfrmt as a column name, use
 #'   quotes around the value: `tfrmt(columnn = "my_object")`.
 #'
+#'   - In a similar vein, when supplying variable names that match `tfrmt`
+#'   argument names, it is recommended to put quotes around the value as such:
+#'   `tfrmt(label = "group")`. In this case, the quoting will prevent `tfrmt`
+#'   from assigning its `group` input value to the `label` value.
```

Comment:
```suggestion
#'   - Additionally, unquoted expressions that match `tfrmt`'s other 
#'   argument names can cause unexpected results. It is recommended 
#'   to put quotes around the value as such:
#'   `tfrmt(label = "group")`. In this case, the quoting will prevent `tfrmt`
#'   from assigning its `group` input value to the `label` value.
```

```suggestion
#'   - Additionally, unquoted expressions that match `tfrmt`'s other argument names can cause unexpected results. 
```
The saying "in a similar vein" is a little colloquial/would potentially require readers to look up that saying is. I think just getting to the point may be more inclusive.